--- accepted-graphics.c
+++ alpha-discard-disabled-graphics.c
@@ -153,7 +153,7 @@
     "out vec4 result;\n"
     "void main() {\n"
     "    vec4 texel = texture(atlas, fragmentUv);\n"
-    "    if (texel.a < 0.5) discard;\n"
+    "    if (texel.a < 0.0) discard;\n"
     "    float fog = clamp((fragmentDistance - fogRange.x) / (fogRange.y - fogRange.x), 0.0, 1.0);\n"
     "    float brightness = min(fragmentLighting.x * light + fragmentLighting.y, 1.0);\n"
     "    result = vec4(mix(texel.rgb * fragmentColor * brightness, fogColor, fog), opacity);\n"
