--- accepted-graphics.c
+++ translucent-depth-write-graphics.c
@@ -719,7 +719,7 @@
     glUniform1f(glGetUniformLocation(world_program, "opacity"), (float)opacity);
     glEnable(GL_BLEND);
     glBlendFunc(GL_SRC_ALPHA, GL_ONE_MINUS_SRC_ALPHA);
-    glDepthMask(GL_FALSE);
+    glDepthMask(GL_TRUE);
     glDisable(GL_CULL_FACE);
     glBindTexture(GL_TEXTURE_2D, texture);
     glBindVertexArray(mesh->array);
